mailbox/elasticsearch/src/main/java/org/apache/james/mailbox/elasticsearch/events/ElasticSearchListeningMessageSearchIndex.java

```diff
@@ -146,10 +146,12 @@ public void delete(MailboxSession session, Mailbox mailbox, List<MessageUid> exp
                 .map(uid ->  indexIdFor(mailbox, uid))
                 .collect(Collectors.toList()));
         } catch (Exception e) {
-            LOGGER.error(String.format("Error when deleting messages %s in mailbox %s from index",
-                mailbox.getMailboxId().serialize(),
-                ImmutableList.copyOf(expungedUids).toString()),
-                e);
+            if (LOGGER.isErrorEnabled()) {
```

Comment:
Maybe we can remove this "isErrorEnabled"

Reply:
Hum I think the copyOf can be a bit expansive, it is good to have a if here

Reply:
**UPDATE:** I have pushed another commit changing the logger call to pass `expungedUids.toArray()` rather than `expungedUids`.

SLF4J supports logging arrays directly, but not Collections. For Collection instances, SLF4J relies on toString(), which will probably produce the same output; however, toString() is not a method of the `List` interface, so its behavior is unspecified and left to implementations.